db/schema.rb

```diff
@@ -53,6 +53,7 @@
     t.string "service"
     t.index ["local_authority_id"], name: "ix_api_users_on_local_authority_id"
     t.index ["name", "local_authority_id"], name: "index_api_users_on_name_and_local_authority_id", unique: true
```

Comment:
Looks like this index already exists but it's not using the prefix `ix_` so rubocop didn't pick it up - update the migration to drop this index if it exists.